specification/newrelic/resource-manager/NewRelic.Observability/preview/2023-10-01-preview/NewRelic.json

```diff
@@ -0,0 +1,2878 @@
+    "AccountInfo": {
+      "type": "object",
+      "properties": {
+        "accountId": {
+          "type": "string",
+          "description": "Account id"
+        },
+        "ingestionKey": {
+          "$ref": "#/definitions/SecureString",
+          "description": "ingestion key of account"
+        },
+        "region": {
+          "type": "string",
+          "description": "NewRelic account region"
+        }
+      },
+      "description": "Account Info of the NewRelic account"
+    },
+    "AccountProperties": {
+      "type": "object",
+      "properties": {
+        "organizationId": {
+          "type": "string",
+          "description": "organization id"
+        },
+        "accountId": {
+          "type": "string",
+          "description": "account id"
+        },
+        "accountName": {
+          "type": "string",
+          "description": "account name"
+        },
+        "region": {
+          "type": "string",
+          "description": "NewRelic account region"
+        }
+      },
+      "description": "List of all the New relic accounts for the given user"
+    },
+    "AccountsListResponse": {
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AccountResource"
+          },
+          "x-cadl-name": "AccountResource[]",
+          "description": "The AccountResource items on this page"
+        },
+        "nextLink": {
+          "type": "string",
+          "format": "uri",
+          "description": "The link to the next page of items",
+          "x-cadl-name": "Rest.ResourceLocation"
+        }
+      },
+      "description": "Response of get all accounts Operation.",
+      "required": [
+        "value"
+      ]
+    },
+    "BillingInfoResponse": {
+      "description": "Marketplace Subscription and Organization details to which resource gets billed into.",
+      "type": "object",
+      "properties": {
+        "marketplaceSaasInfo": {
+          "description": "Marketplace Subscription details",
+          "$ref": "#/definitions/MarketplaceSaaSInfo"
+        },
+        "partnerBillingEntity": {
+          "description": "Partner Billing Entity details: Organization Info",
+          "$ref": "#/definitions/PartnerBillingEntity"
+        }
+      }
+    },
+    "MarketplaceSaaSInfo": {
+      "description": "Marketplace SAAS Info of the resource.",
+      "type": "object",
+      "properties": {
+        "marketplaceSubscriptionId": {
+          "description": "Marketplace Subscription Id. This is a GUID-formatted string.",
+          "type": "string"
+        },
+        "marketplaceSubscriptionName": {
+          "description": "Marketplace Subscription Details: SAAS Name",
+          "type": "string"
+        },
+        "marketplaceResourceId": {
+          "description": "Marketplace Subscription Details: Resource URI",
+          "type": "string"
+        },
+        "marketplaceStatus": {
```

Comment:
>marketplaceStatus [](http://example.com/codeflow?start=9&length=17)

Can this be enum ?

Reply:
We are keeping this is string because partner can keep it their side with different strings, and we will be writing logic based on that only.